TurkishFilter/sections/specific.txt

```diff
@@ -66,6 +66,7 @@ fullfilmcidayim.com,dizispeed.com,babafullfilmizle.com###adStarter
 /service/advertchannels?$domain=urfanatik.com|dirilispostasi.com|turkuazgazetesi.net|yenialanya.com|bugunkocaeli.com.tr|cagdaskocaeli.com.tr|mansetaydin.com|enkocaeli.com|kocaelikoz.com|kocaelibarisgazetesi.com|seskocaeli.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+||play-dizigom1.cdn.ampproject.org/*/oreklam.png
```

Comment:
image is specific to this domain only. `*` is unnecessary
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/167502257-c0cb3160-b199-45a2-ba6f-b0fe91f3be45.png)

</details>